src/pybind/mgr/dashboard/plugins/ses7p_motd.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import hashlib
+import json
+
+from mgr_module import Option
+
+from . import PLUGIN_MANAGER as PM
+from . import interfaces as I  # noqa: E741,N812
+
+
+@PM.add_plugin  # pylint: disable=too-many-ancestors
+class SES7PMotd(I.CanMgr, I.Setupable, I.HasOptions):
+    @PM.add_hook
+    def get_options(self):
+        return [Option(
+            name='ses7p_motd_configured',
+            default=False,
+            type='bool',
+            desc='Is the MOTD for SES7P configured')]
+
+    @PM.add_hook
+    def setup(self):
+        # Check whether the MOTD has already been configured. This is
+        # done to ensure the SES7P announcement is not configured on
+        # every start of the Ceph Dashboard module and to allow the
+        # administrator to disable it if necessary.
+        configured = self.mgr.get_module_option('ses7p_motd_configured')
+        if configured:
+            return
+        # Note, we can not call the 'ceph dashboard motd set ...' command
+        # here because it is not available at this time. To workaround
+        # this issue we build and write the MOTD configuration ourself.
+        message = 'There is an upgrade to SES7P available, please check the ' \
```

Comment:
- We need to update this URL after the SES7 MU (SES7p) documentation is available.
- Shall we use `SES7P` or `SES7 MU` here?

Reply:
Providing a redirect for .../7p/... to .../7.1/... or .../7MU/... is not an issue. (Though it would be great to know the correct product version right from the start.)

Reply:
The URL has been changed to `https://documentation.suse.com/ses/7/html/ses-all/ses-upgrade.html` because SES7p will replace the documentation of SES7.

Reply:
@tbazant

* I guess that means you'll keep using the same maintenance/ branch for SES 7 and SES 7P?

* Also, I am wondering -- Rook and SES/Windows are probably no longer supported for SES 7P, right? We'd still want the associated docs published somehow though, I think. Either in an on-page category (e.g. "Legacy docs") or they should be displayed on an entirely different page (e.g. "SES 7 Legacy"). What do you think?

Reply:
It will be supported and it will remain the same. Rook will also receive an update that will be pushed. If you imagine SES 7 is simply getting a maintenance update - but a large one - that's the only difference here. We do not want customers to realise there's a difference between SES 7 and 7 P. Once 7 P is out it's the only version of SES 7 available.

@ml8mr - SES for Windows will remain the same, si?

Reply:
> It will be supported and it will remain the same. Rook will also receive an update that will be pushed. If you imagine SES 7 is simply getting a maintenance update - but a large one - that's the only difference here. We do not want customers to realise there's a difference between SES 7 and 7 P. Once 7 P is out it's the only version of SES 7 available.

Agreed. Rook actually already supports Ceph Pacific, but it will continue to receive upgrades as necessary.

> @ml8mr - SES for Windows will remain the same, si?

Yes. The SES for Windows installation will remain unchanged, and will work just fine with Ceph Pacific.

Reply:
@votdev We've decided that the upgrade should be called 7.1. Can you switch it to that, so we can get this merged?

Reply:
ping @votdev, can we update this to reference 7.1? See also the image for the upgrade doc https://github.com/SUSE/doc-ses/pull/987

Reply:
@tbazant Is the URL `https://documentation.suse.com/ses/7/html/ses-all/ses-upgrade.html` still correct?

Reply:
@votdev this URL ^^^ referes to both upgrade paths (SES 6 -> 7.1 and SES 7 -> 7.1). To refer to 7 -> 7.1 only, use `https://documentation.suse.com/ses/7/html/ses-all/upgrade-to-pacific.html` (does not exist yet but should be correct).